inference-engine/src/mkldnn_plugin/mkldnn_graph.cpp

```diff
@@ -810,24 +819,33 @@ void MKLDNNGraph::Infer(MKLDNNInferRequest* request, int batch) {
 
     ENABLE_CPU_DEBUG_CAP(NodeDumper nd(config.debugCaps, infer_count));
 
-    for (int i = 0; i < graphNodes.size(); i++) {
-        if (request != nullptr) {
+    for (int i = 0; i < constantGraphNodes.size(); i++) {
+        if (request != nullptr)
+            request->ThrowIfCanceled();
+
+        if (batch > 0)
+            constantGraphNodes[i]->setDynamicBatchLim(batch);
```

Comment:
Does this call make sense for a constant node?

Reply:
Yes, it's useless for constant nodes, I removed.
And now for constant layer we only dump blobs so I added directive `ifdef` with `CPU_DEBUG_CAPS` for this